### DISCUSSION TITLE
我电脑上的Clash Meta像是个DDOS肉鸡。求帮助

### DISCUSSION THREAD

**Discussion by Yeung-Ming** [MetaCubeX/mihomo#1353] [Q&A] [answered]
![1322b7e82ee6b29121cff6f6021f194](https://github.com/MetaCubeX/mihomo/assets/74055211/945a9fac-99b4-4780-969a-0c9e4bd4776a)
如上图显示，Clash Meta把我家的上行带宽全部拉满。大概是7MB每秒到16MB每秒
![屏幕截图 2024-06-30 074616](https://github.com/MetaCubeX/mihomo/assets/74055211/b360c9b4-50f6-4314-bb0c-496507400d48)
日志疯狂刷新，并且大部分都是其他国家和地区的ip地址，他们一直在向我的鸡场节点发送请求

**Comment by Plutonium141** [comment]
这是把内核安在路由器上了吗？

**Reply by nxtreaming** [reply]
这是PC的截图，哪里来的把内核安装在路由器的上说法？

这就是被这个项目当作肉鸡了。

**Reply by Plutonium141** [reply]
> 这是PC的截图，哪里来的把内核安装在路由器的上说法？
> 
> 这就是被这个项目当作肉鸡了。

确实是pc的截图，我看错了，不过他连接相关的截图都没发你怎么就知道是被这个项目当作肉鸡了？

**Reply by TurnOffNOD** [reply]
> 这是PC的截图，哪里来的把内核安装在路由器的上说法？
> 
> 这就是被这个项目当作肉鸡了。

有，openwrt路由器上有openclash项目，就是把clash（和clash meta）内核安装在路由器上的。
虽然lz这里不是。

**Comment by Plutonium141** [comment, marked as answer]
可以先关闭允许局域网连接后重启软件，确保所有连接都切断后，看看问题是否依旧
检查路由器上是不是对电脑开启了dmz，端口映射或者端口转发等功能，如果是应该关掉
还要检查路由器的防火墙等功能是不是关了
然后检查电脑是不是有病毒

**Reply by Yeung-Ming** [reply]
嗯。今天早上我就把防火墙设置为Clash Meta入站端口连接阻止了。就好起来了

**Reply by younijiuhao876** [reply]
所以最后结论是啥

**Comment by TurnOffNOD** [comment]
电脑上其它软件走clash meta代理的话，weindows也会显示是clash meta在上传下载。最后查出来是哪个软件在上传了没？

**Reply by Plutonium141** [reply]
这个应该是把代理端口对外开放了，不是电脑里的软件在上传

**Reply by TurnOffNOD** [reply]
> 这个应该是把代理端口对外开放了，不是电脑里的软件在上传

那么就算不是这台电脑的，那么也是其它设备上面的软件有问题啊。

**Reply by Plutonium141** [reply]
> > 这个应该是把代理端口对外开放了，不是电脑里的软件在上传
> 
> 那么就算不是这台电脑的，那么也是其它设备上面的软件有问题啊。

这个不好说，具体要在clash的连接里看，还有可能是从互联网上连进来的

**Reply by TurnOffNOD** [reply]
> > > 这个应该是把代理端口对外开放了，不是电脑里的软件在上传
> > 
> > 
> > 那么就算不是这台电脑的，那么也是其它设备上面的软件有问题啊。
> 
> 这个不好说，具体要在clash的连接里看，还有可能是从互联网上连进来的

等等，这位仁兄是电脑有公网ip，clash的代理端口使用了默认，然后还对外开放了clash的代理端口？

**Reply by Plutonium141** [reply]
> 等等，这位仁兄是电脑有公网ip，clash的代理端口使用了默认，然后还对外开放了clash的代理端口？

差不多

**Reply by TurnOffNOD** [reply]
> > 等等，这位仁兄是电脑有公网ip，clash的代理端口使用了默认，然后还对外开放了clash的代理端口？
> 
> 差不多

。。。

我一直还以为是家庭宽带路由器拨号，家里所有设备都是内网ip那种……

**Comment by balh55y** [comment]
我也出现了这种情况,给设备DMZ,然后clash默认7890端口就会被作为肉鸡,去ddos攻击一个ip段,通过ip反查域名进去就能看见他们说被攻击了,,,,,,

**Reply by balh55y** [reply]
但是我并没有暴露7890端口,完全搞不懂是怎么通过我的设备去攻击别人的
